lib/litmus_paper/cli/agent_check.rb

```diff
@@ -1,17 +1,28 @@
 require 'optparse'
+require 'litmus_paper/agent_check_server'
 
 module LitmusPaper
   module CLI
     class AgentCheck
 
       def parse_args(args)
         options = {}
+        options[:pid_file] = '/tmp/litmus-agent-check.pid'
         optparser = OptionParser.new do |opts|
-          opts.on("-s", "--service SERVICE", "Service to check") do |s|
-            options[:service] = s
+          opts.on("-s", "--service SERVICE:PORT,...", Array, "agent-check service to port mappings") do |s|
+            options[:services] = s
           end
-          opts.on("-c", "--config CONFIG", "Path to config file") do |c|
-            options[:config] = c
+          opts.on("-c", "--config CONFIG", "Path to litmus paper config file") do |c|
+            options[:litmus_paper_config] = c
+          end
+          opts.on("-p", "--pid-file PID_FILE", String, "Where to write the pid") do |p|
+            options[:pid_file] = p
+          end
+          opts.on("-w", "--workers WORKERS", Integer, "Number of worker processes") do |w|
+            options[:workers] = w
```

Comment:
maybe provide a sane default e.g `1` here if none is passed same with `pid_file` to `/var/run/litmus_agent` or something like that